### DISCUSSION TITLE
User rights error when reconnecting to a paused fabric capacity

### DISCUSSION THREAD

**Discussion by donsvensen** [TabularEditor/TabularEditor3#1419] [🙋♀️TE3 Q&A]
Hi TE3

I have a fabric capacity that is pausing every hour.

When I work on a semantic model (direct lake) and the capacity pauses - and then I restart the capacity via the azure portal - I get this error when trying to reconnect to the model.

![image](https://github.com/user-attachments/assets/d315721f-583d-426b-95af-2ac883b9c7d5)

Restarting the tabular editor doesn't help - but If I reboot my machine I can reconnect to the model.

Any ideas 

Erik

**Comment by otykier** [comment]
Hi Erik,
Is this using the latest version 3.18.2 of Tabular Editor?

**Reply by donsvensen** [reply]
Hi Daniel,

Yes - 
![image](https://github.com/user-attachments/assets/b23785d8-4f9e-4beb-8f9d-31853d1da074)

**Reply by otykier** [reply]
Thanks for confirming. I don't understand why a reboot would help, as TE does not store any state that is cleared upon reboot. Are you sure it's not just a matter of waiting sufficiently long for the capacity to "warm up"?

Did you try both the "Integrated" and "Microsoft Entra MFA" authentication options?

![image](https://github.com/user-attachments/assets/f607265a-e68c-4355-90f8-c92c652330e0)

**Reply by donsvensen** [reply]
You are right - it doesnt help to reboot - (that was yesterday I tried that and It worked )

I can't use integrated mode as the model is running in another tenant.

The capacity has been active for 20 min so I should be warmed up :-) - and I can run reports in the service as well ... and modify the datamodel via the service as well

**Reply by otykier** [reply]
Sounds like it's a permissions issue then. Could you check that the account you're using has at least Workspace Contributor access (i.e. "write" permission on the semantic model itself)?

https://learn.microsoft.com/en-us/power-bi/enterprise/service-premium-connect-tools#security

**Reply by donsvensen** [reply]
I don't think so - because I can do it in the power bi service with same use

**Reply by otykier** [reply]
And you're sure that XMLA read/write access is (still) enabled on the capacity?

**Reply by otykier** [reply]
If you would rather take this offline, feel free to send an e-mail to support@tabulareditor.com
